templates/docs/examples/patterns/tab-buttons/default.html

```diff
@@ -1,7 +1,7 @@
 {% extends "_layouts/examples.html" %}
 {% block title %}Tab Buttons / Default{% endblock %}
```

Comment:
We should update the titles of these to say they are deprecated

Reply:
Something like "Tab Buttons (deprecated) / Default"?